Tools/CISupport/ews-build/steps.py

```diff
@@ -3984,6 +3984,7 @@ def start(self, BufferLogObserverClass=logobserver.BufferLogObserver):
         self.log_observer_json = logobserver.BufferLogObserver()
         self.addLogObserver('json', self.log_observer_json)
         self.setLayoutTestCommand()
+        self.command = self.shell_command(' '.join(self.command) + ' 2>&1 | python3 Tools/Scripts/filter-test-logs layout')
```

Comment:
Curious, can we drop python3 from this command and invoke the filter-test-logs directly?